app/dashboard/_components/CardHeader.tsx

```diff
@@ -0,0 +1,32 @@
+'use client'
+
+import Dropdown from './Dropdown'
```

Comment:
Can't do that for Dropdown since they're in the same file :(

Reply:
@/app/dashboard/_components/DropDown

Reply:
*correction: i don't wanna do that cus they are in the same file. But forgot we agreed to always use @ import because it's hard to now when not to use it.